CHANGELOG.md

```
@@ -1,3 +1,15 @@
+# v2.0.4 (Sat Feb 07 2026)
+
+#### 🐛 Bug Fixes
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)